packages/terminal/extension.package.json

```diff
@@ -10,6 +10,9 @@
     "@types/xterm": "^2.0.3",
     "xterm": "^2.8.1"
   },
+  "devDependencies": {
```

Comment:
Do you really need it? ws should be available via @theia/core

Reply:
Don't I need to specific direct dependencies ?

Reply:
all transitive are available automatically, with the redeclaration we have to manage versions in 2 places

Reply:
OK I guess I have to get used to this... it's not easy with the hoisting you have to manually verify that one of your direct deps includes it... testing it is not enought